devicetypes/smartthings/zll-switch.src/zll-switch.groovy

```diff
@@ -0,0 +1,124 @@
+/**
+ *  Copyright 2016 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+import groovy.transform.Field
+
+@Field Boolean hasConfiguredHealthCheck = false
```

Comment:
I haven't seen this before. Does this actually work? I would think you'd need to store this in `state` or else it wouldn't be persisted across DTH executions.

Reply:
@tpmanley  This is also in the ZLL Dimmer Bulb DTH. I don't have enough knowledge on health check to answer this.